src/frontend/components/RecommendedVideoCard/RecommendedVideoCard.js

```diff
@@ -2,19 +2,23 @@ import React, { useState } from "react";
 import Moment from "react-moment";
 import { nFormatter } from "../../utils/nFormatter";
 import { useNavigate } from "react-router-dom";
-import { useAuth, useWatchLaterVideos } from "../../contexts";
+import { useAuth, useWatchLaterVideos, useHistory } from "../../contexts";
```

Comment:
you can use [Absolute imports](https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8)

Reply:
Oh, I'll check this out.